config.cmake.template

```diff
@@ -50,13 +50,48 @@
 # set(BDDCML_ROOT "$ENV{HOME}/local/BDDCML/bddcml-2.2")
 
 ### Python setting ###############################################################
-# USE_PYTHON - only use python if this is set to "yes"
+# USE_PYTHON - use embedded python if this is set to "yes"
 # We add python library only on explicit request since it leads to tons of errors when debugging with valgrind.
 #
 # set(USE_PYTHON "yes")
 
 # PYTHON_COPY - make copy of python files, necessary for release builds
 # set(PYTHON_COPY "yes")
+#  - Following variables MUST ALSO BE SET:
+#    - PYTHON_COPY_ROOT
```

Comment:
Rename can be done. But hint cannot be used. There are two Python related FindPackages
- [`find_package(PythonLibs)`](https://cmake.org/cmake/help/v3.0/module/FindPythonLibs.html) - which finds include files and libraries (so)
- [`find_package(PythonInterp)`](https://cmake.org/cmake/help/v3.0/module/FindPythonInterp.html) - which interpret binary (e.g. python command)

`PythonLibs` (includes and libs) as well as `PythonInterp` can be outside PYTHON_ROOT